tests/x11/chromium.pm

```diff
@@ -12,43 +12,32 @@ use Mojo::Base 'x11test', -signatures;
 use testapi;
 use utils;
 
-# Prevent lost characters due to temporary unresponsiveness of chromium address bar.
-# It looks like chromium becomes unresponsive after typing about 3
-# characters, likely due to some auto-completion feature.
-# See https://progress.opensuse.org/issues/109737 for details
-sub type_address ($string) {
-    type_string substr($string, 0, 10), @_, max_interval => utils::SLOW_TYPING_SPEED;
-    enter_cmd substr($string, 10), @_;
+# Avoid lost characters due to temporary unresponsiveness by re-launching
+# with the correct URL via the command-line.
+sub launch_with_url ($url, %args) {
+    # avoid async keyring popups
+    # allow key input before rendering is done, see poo#109737 for details
+    x11_start_program('chromium --password-store=basic --allow-pre-commit-input ' . $url, %args);
 }
 
 sub run {
     select_console 'x11';
     mouse_hide;
     ensure_installed 'chromium';
 
-    # avoid async keyring popups
-    x11_start_program('chromium --password-store=basic', target_match => 'chromium-main-window', match_timeout => 50);
-
-    wait_screen_change { send_key 'esc' };    # get rid of popup (or abort loading)
-    send_key 'ctrl-l';    # select text in address bar
-
-    # Additional waiting to prevent unready address bar
-    # https://progress.opensuse.org/issues/36304
-    assert_screen 'chromium-highlighted-urlbar';
-    type_address('chrome://version ');
-    assert_screen 'chromium-about';
-
-    send_key 'ctrl-l';
-    assert_screen 'chromium-highlighted-urlbar';
-    type_address('https://html5test.opensuse.org');
-    assert_screen 'chromium-html-test', 90;
+    launch_with_url('https://html5test.opensuse.org', target_match => 'chromium-html-test');
+    send_key 'alt-f4';
 
     # check a site with different ssl configuration (boo#1144625)
-    send_key 'ctrl-l';
-    assert_screen 'chromium-highlighted-urlbar';
-    type_address('https://upload.wikimedia.org/wikipedia/commons/d/d0/OpenSUSE_Logo.svg');
-    assert_screen 'chromium-opensuse-logo', 90;
+    launch_with_url('https://upload.wikimedia.org/wikipedia/commons/d/d0/OpenSUSE_Logo.svg',
+        target_match => 'chromium-opensuse-logo');
+
+    # the version URL has to be typed since it is ignored on the command-line
+    send_key 'ctrl-l';    # select text in address bar
+    enter_cmd('chrome://version');
+    assert_screen('chromium-about');
```

Comment:
so you are actually going back to typing faster again but hope that it works reliably because you opened chromium in before or why should this work? I suggest to trigger multiple hundreds of jobs for verification. If this does not work in a stable manner I suggest to try use `assert_and_click` as alternative

Reply:
No. Because of --allow-pre-commit-input

Reply:
oh, cool. Where have you found that? Feel welcome to share a bit more in the git commit message :) Still then please at least *one* updated verification run for the happy-path :)

Reply:
https://openqa.opensuse.org/tests/2408670#step/chromium/24